app/services/users/profile.rb

```diff
@@ -52,9 +52,12 @@ def fetch_and_serialize_profile
     end
 
     def account
-      {
-        account_uuid: user.account_uuid
-      }
+      account = Account.cache_or_create_by!(user)
```

Comment:
this looks good 👍one question: do we only want this for the user endpoint? if account_uuid is delegated to the account method could it be cached or fetched there?

Reply:
Good call!

Implemented in https://github.com/department-of-veterans-affairs/vets-api/pull/3048/commits/ea67a0fb53b6d1fc442fae56aef186860f0aecf2